src/stdlib_stats_median.fypp

```diff
@@ -107,17 +108,18 @@ contains
                   end if
                 #:endif
 
-                call sort(x_tmp)
+                call select(x_tmp, c, val)
 
                 if (mod(n, 2) == 0) then
+                    call select(x_tmp, c+1, val1, left = c)
```

Comment:
As above, `minval` is likely faster (below I won't repeat)

Reply:
done